Gemfile

```diff
@@ -58,8 +58,8 @@ group :development do
 end
 
 group :guard do
```

Comment:
Can you remove this commit since is being addressed in https://github.com/metricfu/metric_fu/pull/310 ?

Reply:
@bronzdoc I rebased with `upstream` and now this is solved. 👍